ui/admin/app/routes/scopes/scope/sessions/index.js

```diff
@@ -162,18 +171,32 @@ export default class ScopesScopeSessionsIndexRoute extends Route {
     );
   }
 
+  async canListUsers(orgId) {
+    const orgScope = await this.store.findRecord('scope', orgId);
+    const globalScope = await this.store.findRecord('scope', 'global');
```

Comment:
These two should be passed in rather than fetched so we don't call multiple times. I think using a `canListUsers` is fine but you also could have just wrapped the permissions check as part of `getAssociatedUsers`.

Reply:
Right. That would be much better. I will update that now.